src/mame/drivers/alpha68k.c

```diff
@@ -110,7 +110,6 @@ Stephh's additional notes (based on the games M68000 code and some tests) :
   - There is no Dip Switch to determine if you are allowed to continue a game
     or not, so you ALWAYS have the possibility to continue a game.
 
- 
```

Comment:
Could you please reinstate the extra blank line?  The double blank line separates sections as opposed to the single blank line separating paragraphs in this block comment.